### PR TITLE
Update README and correct lint error

### PR DESCRIPTION
**Pull request by joshcanhelp**
Update README with pre-early access notice and standardize the rest.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,51 @@
-Authentication middleware for express.js that validate access tokens following [RFC 6750](https://tools.ietf.org/html/rfc6750).
+Authentication middleware for Express.js that validates access tokens following [RFC 6750](https://tools.ietf.org/html/rfc6750). The purpose of this library is to protect Oauth2 resources.
```
Comment (lbalmaceda) [inline README.md]: Small suggestion. The RFC refers to it as "OAuth 2.0". 

```suggestion
Authentication middleware for Express.js that validates access tokens following [RFC 6750](https://tools.ietf.org/html/rfc6750). The purpose of this library is to protect OAuth 2.0 resources.
```

File: README.md [hunk 2]
```diff
@@ -1,36 +1,51 @@
...
-## Usage
+## Getting Started
 
-If the access token received by your API is a token signed in the same way than OpenID Connect id_token, you can use the openid strategy as follows:
+The library needs the following values to authroize requests:
 
-```javascript
-const { auth, strategies, requiredScopes } = require('express-oauth2-bearer');
+- **Issuer Base URL**: The base URL of the authorization server. If you're using Auth0, this is your tenant **Domain** pre-pended with `https://` (like `https://tenant.auth0.com`) found on the **Settings** tab for your Application in the [Auth0 dashboard](https://manage.auth0.com).
+- **Allowed Audiences**: Audience identifier (or mulitple separated by a comma) allowed for the access token. If you're using Auth0, this is the **Identifier** found on the **Settings** tab for your API in the [Auth0 dashboard](https://manage.auth0.com/#/apis).
```
Comment (lbalmaceda) [inline README.md]:
```suggestion
- **Allowed Audiences**: Audience identifier (or multiple separated by a comma) allowed for the access token. If you're using Auth0, this is the **Identifier** found on the **Settings** tab for your API in the [Auth0 dashboard](https://manage.auth0.com/#/apis).
```

File: README.md [hunk 3]
```diff
@@ -66,23 +81,65 @@ app.get('/products',
   });
 ```
 
+### API Documentation:
+
+`auth()` accepts an asynchronous function receiving a token and returning a set of claims.
```
Comment (lbalmaceda) [inline README.md]: is it worth clarifying _which_ token?

File: README.md [hunk 4]
```diff
@@ -66,23 +81,65 @@ app.get('/products',
...
+
+`requiredScopes()` accepts either an string or an array of strings.
```
Comment (lbalmaceda) [inline README.md]:
```suggestion
`requiredScopes()` accepts either a string or an array of strings.
```

File: README.md [hunk 5]
```diff
@@ -66,23 +81,65 @@ app.get('/products',
...
+
+`strategies.openid` accepts the following parameters:
+
+
+| Name                | Default                            | Description                                                          |
+|---------------------|------------------------------------|----------------------------------------------------------------------|
+| issuerBaseURL       | `env.ISSUER_BASE_URL`              | URL for the token issuer.                                            |
+| allowedAudiences    | `env.ALLOWED_AUDIENCES.split(',')` | Allowed audiences for the token.                                     |
+| clockTolerance      | `5`                                | Clock tolerance in seconds for token verification, aka leeway.       |
```
Comment (lbalmaceda) [inline README.md:97]: Any particular reason to use 5 secs over other value? e.g. auth0-spa-js default leeway is 60.

File: README.md [hunk 6]
```diff
@@ -66,23 +81,65 @@ app.get('/products',
...
+| clientSecret        | `env.CLIENT_SECRET`                | Cleitn secret, required for tokens signed with symmetric algorithms. |
```
Comment (lbalmaceda) [inline README.md]:
```suggestion
| clientSecret        | `env.CLIENT_SECRET`                | Client secret, required for tokens signed with symmetric algorithms. |
```

File: README.md [hunk 7]
```diff
@@ -66,23 +81,65 @@ app.get('/products',
...
+
+When you're ready to push your changes, please run the lint command first:
+
+```bash
+npm run lint
+```
+
+## Support + Feedback
+
+Please use the [Issues queue](https://github.com/auth0/express-openid-connect/issues) in this repo for questions and feedback.
```
Comment (lbalmaceda) [inline README.md]: wrong repo link?

```suggestion
Please use the [Issues queue](https://github.com/auth0/express-oauth2-bearer/issues) in this repo for questions and feedback.
```

**Review by lbalmaceda — CHANGES_REQUESTED** [review]

File: README.md [hunk 8]
```diff
@@ -66,23 +81,65 @@ app.get('/products',
...
+## Contributing
+
+We appreciate feedback and contribution to this repo! Before you get started, please see the following:
+
+- [Auth0's general contribution guidelines](https://github.com/auth0/.github/blob/master/CONTRIBUTING.md)
+- [Auth0's code of conduct guidelines](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md)
+
+Contributions can be made to this library through PRs to fix issues, improve documentation or add features. Please fork this repo, create a well-named branch, and submit a PR with a complete template filled out.
+
+Code changes in PRs should be accompanied by tests covering the changed or added functionality. Tests can be run for this library with:
```
Comment (lbalmaceda) [inline README.md:109]: Maybe this should be part of a `DEVELOPING` file in the root

Comment (joshcanhelp) [reply README.md:109]: At some point, maybe ... just wanted to get this information in there for now.

File: README.md (hunk 5 above)
Comment (joshcanhelp) [reply README.md:97]: That's just what it came with. Would need to update it in the code as well. We'll make that part of the EA criteria.

File: README.md (hunk 3 above)
Comment (joshcanhelp) [reply README.md]:
```suggestion
`auth()` accepts an asynchronous function receiving an access token and returning a set of claims.
```

**Comment by joshcanhelp** [comment]
@lbalmaceda - Ready for you when you have a sec (not sure if you got the re-review notification), not urgent!

**Comment by lbalmaceda** [comment]
I did!

**Review by lbalmaceda — APPROVED** [review]

**Pull request merged**